### PR TITLE
Toponaming: Match LS3 opcodes for sketch shape

### PR DESCRIPTION
**Pull request by bgbsww**
This fixes a category of TNP bugs that occur when you split the edge in a sketch ( which completely renumbers the geometry ) that is being used to create faces that are referenced (attached to) by something else.

Brings our code and opcode tagging into line with LS3 by placing the sketch opcode on the wire(s) inside the compound instead of on the compound containing the wires.

This then allows the comment about use in getElementName to apply.

Also confirms clause to force map creation is required by our restructuing, so TODO removed.

Basically these are updating early-TNP process code to final state matching LS3.

### CODE REVIEW

**Comment by yorikvanhavre** [comment]
backported
